apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.IntegrationTests/ErrorReporting/ExceptionLoggerTest.cs

```diff
@@ -25,60 +24,49 @@
 using System.Web.Http;
 using System.Web.Http.ExceptionHandling;
 using Xunit;
+using Google.Cloud.Diagnostics.Common;
+using Google.Cloud.Diagnostics.Common.IntegrationTests;
 
 namespace Google.Cloud.Diagnostics.AspNet.IntegrationTests
 {
     public class ExceptionLoggerTest
     {
-        private static string ProjectId;
-        private static string TestId;
+        private static string _projectId;
```

Comment:
moved to private set.  This test is a little hacky how it shares state. I should update this test to match other tests soon